code401challenges/src/main/java/code401challenges/utilities/MultiBracketValidation.java

```diff
@@ -0,0 +1,24 @@
+package code401challenges.utilities;
+
+import code401challenges.stacksandqueues.Stack;
+
+import java.util.ArrayList;
+
+public class MultiBracketValidation {
+    ArrayList<String> brackets = new ArrayList<>();
+
+    public  static  boolean multiBracketValidation(String input) {
+        if(input.length() == 0) {
+            throw new IllegalArgumentException("String length should be longer than 0");
+        }
+        if((input.length() % 2) != 0) {

```

Comment:
The input length doesn't have to be even in order for this method to need to return true. If the input is `{odd}` the method should still return true